src/app/components/generate/generate.component.html

```diff
@@ -0,0 +1,8 @@
+<h1>Generate</h1>
+<div>Name & Type</div>
+<input type="text" [(ngModel)]="name">
+<select [(ngModel)]="type">
+    <option [value]="none" selected>Choose a Class</option>
```

Comment:
I don't like this solution. I would like to have a default selection, should be the most used one (component?)